lib/consumer.ex

```diff
@@ -43,8 +43,7 @@ defmodule Mississippi.Consumer do
 
     children = [
       {ExRabbitPool.PoolSupervisor,
-       rabbitmq_config: amqp_consumer_options,
-       connection_pools: [events_consumer_pool_config(connection_number)]},
+       rabbitmq_config: amqp_consumer_options, connection_pools: [events_consumer_pool_config(connection_number)]},
```

Comment:
just fyi, looks like styler's maximum line length is set to [122](https://github.com/adobe/elixir-styler/blob/95fba171b1ddc945c4ae624f3b3d7e7b425f41a1/lib/styler.ex#L95)